css/css-backgrounds/border-image-beats-transparent-border-color-ref.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>
+  border-image should beat transparent border-color
+</title>
+<style>
+    div {
+        width: 200px;
+        height: 200px;
+        margin: 20px;
+        background-color: silver;
+        border-image-source: linear-gradient(blue, orange);
```

Comment:
I think it would be best to test without a gradient. Is there a shared image resource you can use?